js/pages/SeatPage.ts

```diff
@@ -126,6 +128,22 @@ export class SeatPage extends PageBase {
 
             return ct.nextGameTypeInformation();
         }, this);
+        this.roundNotificationCaption = ko.computed(() => {
```

Comment:
This is very likely should be removed due to #45 and instead of make change in the UI in such way, that message DOM element would be inside TableView HTML template.